centos2ol.sh

```diff
@@ -303,11 +303,25 @@ case "$os_version" in
         fi
         ;;
     8*)
-        # Workaround for qemu-guest-agent packages installed from virt modules
-        if rpm -q qemu-guest-agent; then
-            dnf module reset -y virt
-            dnf module enable -y virt
-            dnf install -y --allowerasing qemu-guest-agent
+        # There are a few dnf modules that are named after the distribution
+        #  for each steam named 'rhel' or 'rhel8' perform a module reset and install
+        mapfile -t modules_enabled < <(dnf module list --enabled | grep rhel | cut -f1 -d\  )
```

Comment:
Would you be open to replacing the `cut` with `awk '{print $1}'` only to remove the "`-d\ `" which I can just see me "fixing" in future because I'm generally blind to whitespace-as-code. 🤓